spec/integrations/shared/base_render_examples.rb

```diff
@@ -79,28 +79,55 @@ def extract(field_name, object, _local_options, _options={})
       obj[:first_name] = nil
     end
 
-    context "Given default value is not provided" do
-      let(:result) { '{"first_name":null,"id":' + obj_id + '}' }
-      let(:blueprint) do
-        Class.new(Blueprinter::Base) do
-          field :id
-          field :first_name
+    context "Given global default field value is specified" do
+      before { Blueprinter.configure { |config| config.field_default = "N/A" } }
+      after { reset_blueprinter_config! }
+
+      context "Given default field value is not provided" do
+        let(:result) { '{"first_name":"N/A","id":' + obj_id + '}' }
+        let(:blueprint) do
+          Class.new(Blueprinter::Base) do
+            field :id
+            field :first_name
+          end
+        end
+        it('global default value is rendered for nil field') { should eq(result) }
+      end
+
+      context "Given default field value is provided" do
+        let(:result) { '{"first_name":"Unknown","id":' + obj_id + '}' }
+        let(:blueprint) do
+          Class.new(Blueprinter::Base) do
+            field :id
+            field :first_name, default: "Unknown"
```

Comment:
Could we switch this example to be `nil` like the one we discussed above. Or alternatively add another test specifically for that functionality.

Reply:
Yeah that's a great suggestion. I'll add another test case for that 👍